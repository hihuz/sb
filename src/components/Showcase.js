import React from "react";

const Showcase = () => (
  <div className="showcase">
    <a className="project" href="https://github.com/hihuz/meny">
      <div className="project-infos">
        <h2 className="project-name">Meny</h2>
        <p className="project-desc">
          A recipe sharing / editing app, work in progress
        </p>
        <div className="project-logos">
          <div className="project-tech">
            <img src="../public/react.png" alt="react" />React
          </div>
          <div className="project-tech">
            <img src="../public/redux.png" alt="redux" />Redux
          </div>
          <div className="project-tech">
            <img src="../public/firebase.png" alt="firebase" />Firebase
          </div>
        </div>
      </div>
    </a>
    <a className="project" href="https://github.com/hihuz/bomb-cleaner">
      <div className="project-infos">
        <h2 className="project-name">Bomb-Cleaner</h2>
        <p className="project-desc">
          A Mine-Sweeper game clone
        </p>
        <div className="project-logos">
          <div className="project-tech">
            <img src="../public/react.png" alt="react" />React
          </div>
          <div className="project-tech">
            <img src="../public/redux.png" alt="redux" />Redux
          </div>
        </div>
      </div>
    </a>
    <a className="project" href="https://github.com/hihuz/sb">
      <div className="project-infos">
        <h2 className="project-name">sb</h2>
        <p className="project-desc">
          The code for this very web site
        </p>
        <div className="project-logos">
          <div className="project-tech">
            <img src="../public/react.png" alt="react" />React
          </div>
          <div className="project-tech">
            <img src="../public/preact.png" alt="preact" />Preact
          </div>
        </div>
      </div>
    </a>
  </div>
);

export default Showcase;
