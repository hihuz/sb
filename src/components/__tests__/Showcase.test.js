import React from "react";
import { shallow } from "enzyme";
import { shallowToJson } from "enzyme-to-json";
import Showcase from "../Showcase";

describe("Showcase", () => {
  test("Snapshot", () => {
    const component = shallow(<Showcase projects={[]} />);
    const tree = shallowToJson(component);
    expect(tree).toMatchSnapshot();
  });
});
