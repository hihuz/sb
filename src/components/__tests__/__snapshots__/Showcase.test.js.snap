// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Showcase Snapshot 1`] = `
<div
  className="showcase"
/>
`;
